lib/ansible/cli/doc.py

```diff
@@ -781,7 +789,7 @@ def run(self):
             elif plugin_type == 'role':
                 docs = self._create_role_list()
             else:
-                docs = self._list_plugins(plugin_type, content)
+                docs = self._list_plugins(plugin_type, content, include_private=(context.CLIARGS['verbosity'] > 0))
```

Comment:
I would rather not overload verbosity for something that is shown with `display.display()` rather than `display.v()`.

Reply:
I got inspired to only show these when `-v` is specified by https://github.com/ansible/ansible/pull/75116#discussion_r997335620. But I'm happy with not doing that :)

Reply:
Maybe that could be a bit surprising too :thinking: